events/index.html

```diff
@@ -8,33 +8,57 @@
         <!-- begin Events -->
         <hr class="whitespace">
         <div id="community-posts" class="row">
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-star" style="color: #df283f;"></i> November 11, 2019</h6>
+                <h2><a href="omero-users-workshop-hamilton-november-2019.html">OMERO Workshop, Hamilton</a></h2>
+                <p class="card-caption">A one-day workshop on how to use OMERO.</p>
+            </div>
+            <hr class="medium-8">
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-star" style="color: #df283f;"></i> October 09-10, 2019</h6>
+                <h2><a href="omero-users-workshop-freising-october-2019.html">OMERO Workshop, Freising</a></h2>
+                <p class="card-caption">A two-day workshop on how to use OMERO.</p>
+            </div>
+            <hr class="medium-8">
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-star" style="color: #df283f;"></i> September 30-October 1, 2019</h6>
```

Comment:
we use 09 and 05 above and below so maybe 01 here (if we want the 0-prefixes at all)